dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/events/event/AbstractEventService.java

```diff
@@ -1855,6 +1890,24 @@ private ProgramInstance getProgramInstance( String uid )
         return programInstance;
     }
 
+    private User getUser( String uid )
```

Comment:
getAssignedUser?

Reply:
in this specific case I think getUser is more appropriate, because although its only used for getting assigned users at the moment, there is no assigned-user specific logic, so it can be used for getting users for  other purposes as well. It is also only part of the preheating/caching :)